apps/sms/views/conversation/js/conversation.js

```diff
@@ -407,6 +407,7 @@ var ConversationView = {
   },
 
   getIdIterator: function conv_getIdIterator() {
+    // TODO: Will need to replace with Conversation service method.
```

Comment:
I think we'll have the method in ConversationService to get all messages, but that we'll have a local array as well; but I'm not sure of course.